CSharp_Course_Project/Task3/Feeder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading;
+
+namespace CSharp_Course_Project.Task3
+{
+    static class Feeder
+    {
+        public static void Feed()
```

Comment:
Please don't use static classes, methods etc.
All we are developing is about OOD.

Reply:
fixed